modules/update-service-install-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-install-web-console_{context}"]
+= Installing the OpenShift Update Service Operator by using the web console
+
+You can use the {product-title} web console to install the OpenShift Update Service Operator.
+
+.Procedure
+
+To install the OpenShift Update Service Operator using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+
+. Choose *OpenShift Update Service* from the list of available Operators, and click *Install*.
+
+[NOTE]
+====
+Enter `Update Service` into the *Filter by keyword...* field to find the operator faster.
+====
+
+. `v1` will be selected as the *Update Channel* since it is the only channel available in this release.
+
+. Select *A specific namespace on the cluster* under *Installation Mode*.
+
+. Select a namespace for *Installed Namespace* or accept the recommended namespace `openshift-update-service`.
+
+. Select an *Approval Strategy*:
++
+** The *Automatic* strategy allows Operator Lifecycle Manager (OLM) to automatically update the Operator when a new version is available.
++
+** The *Manual* strategy requires a cluster administrator to approve the Operator update.
+
+. Click *Install*.
+
+. Verify that the OpenShift Update Service Operator is installed by switching to the *Operators* -> *Installed Operators* page.
+
+. Ensure that *OpenShift Update Service* is listed in the selected namespace with a *Status* of *Succeeded*.
```

Comment:
Please pay attention to the indention and the number. If this block is the sub item of `Choose *OpenShift Update Service* from the list of available Operators, and click *Install*.`, indention is required, or else the number needs to continue from 3.